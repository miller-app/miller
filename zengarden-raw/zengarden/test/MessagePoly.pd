#N canvas 672 297 450 301 10;
#X obj 134 38 loadbang;
#X obj 134 68 t b b b b b;
#X obj 135 167 poly 3 1;
#X msg 239 109 64 60;
#X obj 94 35 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 136 203 pack 0 0 0;
#X obj 136 233 print;
#X msg 72 115 clear;
#X msg 16 131 stop;
#X msg 184 110 58 64;
#X msg 134 110 42 64;
#X msg 295 109 62 64;
#X msg 352 109 60 64;
#X connect 0 0 1 0;
#X connect 1 0 10 0;
#X connect 1 1 9 0;
#X connect 1 2 3 0;
#X connect 1 3 11 0;
#X connect 1 4 12 0;
#X connect 2 0 5 0;
#X connect 2 1 5 1;
#X connect 2 2 5 2;
#X connect 3 0 2 0;
#X connect 4 0 1 0;
#X connect 5 0 6 0;
#X connect 7 0 2 0;
#X connect 8 0 2 0;
#X connect 9 0 2 0;
#X connect 10 0 2 0;
#X connect 11 0 2 0;
#X connect 12 0 2 0;
