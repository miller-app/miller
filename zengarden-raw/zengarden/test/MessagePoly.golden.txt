[@ 0.000ms] print: 1 60 64
[@ 0.000ms] print: 2 62 64
[@ 0.000ms] print: 3 64 60
[@ 0.000ms] print: 1 60 0
[@ 0.000ms] print: 1 58 64
[@ 0.000ms] print: 2 62 0
[@ 0.000ms] print: 2 42 64